File: java/8kyu/Find the Slope.java
// https://www.codewars.com/kata/55a75e2d0803fea18f00009d

public class Slope
{

  public String slope(int[] points)
    {
        int deltaY = points[3] - points[1];
        int deltaX = points[2] - points[0];
        if (deltaX == 0) {
          return "undefined";
        }
        return String.valueOf(deltaY / deltaX);
		}

}

File: javascript/8kyu/Find the Slope.js
// https://www.codewars.com/kata/55a75e2d0803fea18f00009d

function slope(points){
	let x = (points[2] - points[0]);
	let y = (points[3] - points[1]);
	let res = y / x;
	if( x === 0) {
		return 'undefined' ;
	}
	return res + '';
}